public/v4/apps/languagetool.yml

```diff
@@ -4,7 +4,7 @@ services:
         image: erikvl87/languagetool:$$cap_version
         restart: always
         ports:
-            - $$cap_port:8010
+            - 8010:8010
```

Comment:
Thanks for making the change, but you don't need port mapping as long as you have the containerHttpPort